samples/bluetooth/broadcast_audio_sink/Kconfig

```
@@ -28,4 +28,11 @@ config SCAN_DELAY
 	  Time to advertise connectable for a Broadcast Assistant to connect
 	  before starting scanning for Broadcast Sources.
 
+config SD_RECEIVE_STATE_OVERWRITE_OLDEST
+	bool "Overwrite the oldest receive state not being synced"
```

What about already synced ones? 

```suggestion
	bool "Overwrite the inactive oldest receive state."
```